FFXIVClientStructs/FFXIV/Client/Game/InstanceContent/InstanceContentDeepDungeon.cs

```diff
@@ -1,5 +1,12 @@
 namespace FFXIVClientStructs.FFXIV.Client.Game.InstanceContent;
 
+// Client::Game::InstanceContent::InstanceContentDeepDungeon
+//   Client::Game::InstanceContent::InstanceContentDirector
+//     Client::Game::InstanceContent::ContentDirector
+//       Client::Game::Event::Director
+//         Client::Game::Event::LuaEventHandler
+//           Client::Game::Event::EventHandler
+// ctor "E9 ?? ?? ?? ?? 3D ?? ?? ?? ?? 77 73 3D"
```

Comment:
I would suggest using direct signature for InstanceContent ctor, because it is likely to be invalid/wrong if SE adds new stuff to the function where they construct or the compiler optimization
The signature I made from 6.50 binary
``48 89 5C 24 ?? 48 89 6C 24 ?? 48 89 74 24 ?? 57 48 83 EC ?? 48 8B F9 E8 ?? ?? ?? ?? C6 87``